data/specials.inc

```diff
@@ -536,3 +536,6 @@ gSpecials::
 	def_special TrySetBattleTowerLinkType
 	def_special SetTotemBoost
 	def_special TrySpecialOverworldEvo
+    def_special GetNumberSprayStrength
+    def_special GetSprayId
+    def_special GetLastUsedSprayType
```

Comment:
Tabs instead of spaces.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/3594/commits/8e66726ad87290910c8d8beef5fab22b7cfdf5c2